reports/management/commands/import-historical.py

```diff
@@ -8,25 +8,22 @@
 
 class Command(BaseCommand):
     help = 'Import historical GMMP data from an XLSX file'
-    option_list = BaseCommand.option_list + (
-        make_option('--global',
-            action='store_true',
-            help='Coverage is global'),
-        make_option('--region REGION',
-            action='store',
-            dest='region',
-            help='Import historical data for a region. One of: %s' % ', '.join(sorted(REGION_COUNTRY_MAP.keys()))),
-        make_option('--country COUNTRY',
-            action='store',
-            dest='country',
-            help='Import historical data for a country. One of: %s' % ', '.join(sorted(c[0] for c in get_countries()))),
-    )
+    def add_arguments(self, parser):
+        parser.add_argument('--filename', action='append', dest='filename', help='XLSX file to import')
+        parser.add_argument('--global', action='store_true', help='Coverage is global')
+        parser.add_argument('--region REGION', action='store', dest='region', help='Import historical data for a region. One of: %s' % ', '.join(sorted(REGION_COUNTRY_MAP.keys())))
+        parser.add_argument('--country COUNTRY', action='store', dest='country', help='Import historical data for a country. One of: %s' % ', '.join(sorted(c[0] for c in get_countries())))
+
 
     def handle(self, *args, **options):
         historical = Historical()
         coverage = None
         region = None
         country = None
+        filenames = options['filename']
```

Comment:
Why does `filename` end up being `filenames`?

Reply:
I think this comes with a possibility that there might be multiple filenames attached. Saying this because of the `loop` on https://github.com/CodeForAfrica/gmmp/blob/392da892588f6c5b15d9d90e5b16ee2ad2a27f04/reports/management/commands/import-historical.py#L55

Reply:
See my comment here https://github.com/CodeForAfrica/gmmp/pull/113#discussion_r533281050